### PR TITLE
Moved choose mode and choose purpose buttons to diary list screen

### PR DESCRIPTION
**Pull request by sunil07t**

### CODE REVIEW

File: www/js/diary/list.js [hunk 1]
```diff
@@ -374,10 +374,113 @@ angular.module('emission.main.diary.list',['ui-leaflet',
         readAndUpdateForDay(nextDay);
     };
 
-    $scope.toDetail = function() {
-      $state.go('root.main.detail');
+    $scope.toDetail = function(param) {
```
Comment (shankari) [inline www/js/diary/list.js]: why do you need this change?

File: www/templates/diary/list.html [hunk 2]
```diff
@@ -33,17 +33,18 @@
         MUST test on android first to make sure that the scrolling works.
         -->
 
-		<div collection-repeat="tripgj in data.currDayTripWrappers" item-height="differentCommon(tripgj)? '348px':'308px'">
+		<div collection-repeat="tripgj in data.currDayTripWrappers" item-height="differentCommon(tripgj)? '413px':'365px'">
            <div class="start-time-tag">{{localTimeString (tripgj.data.properties.start_local_dt)}}</div>
             <div  style="padding-left: 19%;">
 
-			 <ion-item style="background-color: transparent;" id="no-border" class="list-item" href="#/root/main/diary/{{tripgj.data.id}}">
```
Comment (shankari) [inline www/templates/diary/list.html]: why remove the `href` from here?

Comment (sunil07t) [inline www/templates/diary/list.html]: The buttons are inside `ion-item` so I had to make separate div for buttons and the redirect. If both are just inside `ion-item` the click will direct to trip details and the buttons will not work properly.

Comment (shankari) [inline www/templates/diary/list.html]: can't you leave the href on the non-button div? Just curious so I can understand how href works.

**Comment by sunil07t** [comment]
Changes:
- Moved the choose mode and choose purpose buttons to diary list card. 
- The popover buttons are little bigger than last version so they are easier to press.
- Had to change the height of the cards to show the buttons. 
- After one of the options from the list is chosen the popover will disappear.

Testing done: 
- The popover shows up on the diary list screen
- Spacing and height of the cards in the diary screen are responsive
screen shot 2017-07-29 at 4 20 34 pm
screen shot 2017-07-29 at 4 20 44 pm
screen shot 2017-07-29 at 4 20 52 pm

File: www/templates/diary/list.html (hunk 2 above)
Comment (sunil07t) [inline www/templates/diary/list.html]: Actually, the href did not work when I tested it. So I changed it to `stage.go()` with params

Comment (shankari) [inline www/templates/diary/list.html]: so you left the href on the part of the item that does not have the button?
e.g. `<div ng-click="toDetail(tripgj.data.id)">` ->  `<div href="#/root/main/diary/{{tripgj.data.id}}">`
doesn't work?

Comment (sunil07t) [inline www/templates/diary/list.html]: This does not work

**Pull request merged**